packages/core/content-manager/server/src/services/utils/configuration/metadatas.ts

```diff
@@ -103,11 +103,6 @@ async function syncMetadatas(configuration: any, schema: any) {
     const attr = schema.attributes[key];
 
     const updatedMeta = { edit, list };
-    if (key === 'documentId') {
-      _.set(updatedMeta, ['edit'], {});
-      _.set(acc, [key], updatedMeta);
-      return acc;
-    }
```

Comment:
ℹ️ Locally this change had the tests passing, but we shouldn't need this anyway since we set it in the `createDefaultMetadatas` above.  Which we merge with updatedMetas.